src/devices/Arduino/samples/CharacterDisplay.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Device.Gpio;
+using System.Globalization;
+using System.Text;
+using Iot.Device.CharacterLcd;
+
+namespace Iot.Device.Arduino.Sample
+{
+    internal sealed class CharacterDisplay : IDisposable
```

Comment:
consider creating a separate demo for each of those (i.e. one simple project per file)